packages/core/src/external/aws/lambda-logic/bundle-to-html.ts

```diff
@@ -1022,6 +1038,34 @@ function createConditionSection(conditions: Condition[]) {
 
       return acc;
     }, [] as RenderCondition[])
+    // logic to filter out conditions that are duplictates in all but date, and throw away the ones that dont have dates.
+    .reduce((acc, condition) => {
+      const conditionText = condition.name;
+      const conditionCode = condition.code;
+      const conditionDate = condition.firstSeen;
+
+      if (conditionText === undefined || conditionCode === undefined) {
+        return acc;
+      }
+
+      const existingCondition = acc.find(existingCondition => {
+        const existingConditionText = existingCondition?.name;
+        const existingConditionCode = existingCondition.code;
```

Comment:
Why question mark on `existingCondition?.name` but not `existingCondition.code`? Missed it?

Reply:
no need for question mark on either. good catch